### PR TITLE
Make the service-ca upgrade case can be run in disconnected env

### PR DESCRIPTION
**Pull request by xingxingxia**
This PR fixes OCPQE-4544, the case uses an image not in the mirror list, will fail in disconnected env, see its failure there.

Jenkins is down. Local test passes pre-upgrade part. **Need Jenkins** to run post-upgrade part tomorrow and ask @yaoli-redhat @wangke19 to review then, otherwise the pre-upgrade part will delete the project created if run in **local** `cucumber features/upgrade/api_auth/upgrade.feature:$LINE`, this will make post-upgrade fail to get that project, @geliu2016 @JianLi-RH , how to locally run upgrade case to avoid this problem?

### CODE REVIEW

**Comment by JianLi-RH** [comment]
I am also facing this problem, waiting for the answer:) @xingxingxia @jhou1 @geliu2016

**Comment by xingxingxia** [comment]
Hi, all,
I got it from upgrade_CI job logs' Runner jobs' parameters, must use below extra fields:
```
environments:
  ocp4:
    version: "4.8"
    admin_creds_spec: "..."
    user_manager: UpgradeUserManager
    static_users_map:
      upuser1: "XXX:XXX"
      upuser2: "XXX:XXX"
```
Before-upgrade passed logs: Runner/119260/console (TEST_MODE field: upgrade-prepare-all)
Post-upgrade passed logs: Runner/119272/console (TEST_MODE field: upgrade-check-all)
@yaoli-redhat , @wangke19 , please help review when free, thanks!

File: features/upgrade/api_auth/upgrade.feature [hunk 1]
```diff
@@ -236,47 +237,40 @@ Feature: apiserver and auth related upgrade check
...
+      | openssl x509 -noout -text -in <(echo '<%= File.read("tls.crt") %>') |
     Then the step should succeed
     And the output should contain:
       | DNS:foo.service-ca-upgrade.svc, DNS:foo.service-ca-upgrade.svc.cluster.local |
 
   # @author scheng@redhat.com
   # @case_id OCP-41198
+  @admin
   @upgrade-check
   @users=upuser1,upuser2
```
Comment (yaoli-redhat) [inline features/upgrade/api_auth/upgrade.feature:258]: why need two users? Don't see the scenario needs to switch between upuser1 and upuser2

Comment (xingxingxia) [reply features/upgrade/api_auth/upgrade.feature:258]: Our CI pipeline's standard usage. You can check all other **upgrade** cases that used normal user, they all follow above.

**Comment by wangke19** [comment]
> Hi, all,
> I got it from upgrade_CI job logs' Runner jobs' parameters, must use below extra fields:
> 
> ```
> environments:
>   ocp4:
>     version: "4.8"
>     admin_creds_spec: "..."
>     user_manager: UpgradeUserManager
>     static_users_map:
>       upuser1: "XXX:XXX"
>       upuser2: "XXX:XXX"
> ```
> 
> Before-upgrade passed logs: Runner/119260/console (TEST_MODE field: upgrade-prepare-all)
> Post-upgrade passed logs: Runner/119272/console (TEST_MODE field: upgrade-check-all)
> @yaoli-redhat , @wangke19 , please help review when free, thanks!

@xingxingxia For the before/after-upgrade results, I found 9 times to `oc get nodes <node> --output=yaml` when ran `When I store the ready and schedulable masters in the clipboard `. I think no need so many times to check nodes.

**Comment by xingxingxia** [comment]
@pruan-rht @liangxia @JianLi-RH @dis016 help merge, thanks

**Comment by xingxingxia** [comment]
@pruan-rht @liangxia @JianLi-RH @dis016 could you help merge, thanks

**Review by pruan-rht — COMMENTED** [review]
/lgtm

**Pull request merged**